controllers/backend/volume/volume.go

```diff
@@ -55,3 +57,43 @@ func (v *volumeController) sync(key string, pvc *v1.PersistentVolumeClaim) error
 	_, err = v.volumes.Update(newVol)
 	return err
 }
+
+func (v *volumeController) Create(pvc *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaim, error) {
+	if pvc == nil {
+		return pvc, nil
+	}
+
+	if pvc.Labels["rio.cattle.io/use-templates"] == "" {
```

Comment:
can this be "rio.catle.io/volumetemplate"?  We only put that label on PVC created from VolumeClaimTemplates